### DISCUSSION TITLE
client side Middleware is never execute

### DISCUSSION THREAD

**Discussion by sarkaburanska** [nuxt/nuxt#28308] [Questions]
Hi, I try to use middleware on client side, just easy example of 'XXX.ts' in /middleware defineNuxtRouteMiddleware... and use it in page

definePageMeta({
  middleware: 'XXX',
});
but its never called at all. It seems as the auto-registration doesn't work or something else. Any ideas?

**Comment by AloisSeckar** [comment]
Like this it seems impossible to guess... Can you please provide a reproduction of your code? For example by adding into [StackBlitz Nuxt starter](https://stackblitz.com/github/nuxt/starter/tree/v3-stackblitz?file=README.md)

I have this working example of middleware in Nuxt - https://github.com/AloisSeckar/demos-nuxt/tree/main/nuxt-middleware - so maybe you can check and compare, if you really do the same?
